e2e/components/ProgressIndicator/ProgressIndicator-test.avt.e2e.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('ProgressIndicator', () => {
```

Comment:
I think this is still missing a ".ToHaveNoACViolations(...)" check for each of the states: hover, active, visited. So from my understanding you'd perform the keyboard/page operations to get the component to a "hover" or "active" state, and then you'd run an avt assertion against that. I could be misunderstanding though... 😅

Reply:
@francinelucca At first I tought the same. However I was not sure about it, because there is no keyboard navigation in the default stories, that's why I only tested the interactive story. 

But I think you are right! I'm gonna add those tests.

For example:

![Screenshot 2023-07-11 at 17 11 29](https://github.com/carbon-design-system/carbon/assets/52183462/eff28fdd-85c1-40bd-a1d8-c289000601ab)

```  
test('accessibility-checker on hover @avt', async ({ page }) => {
    await visitStory(page, {
      component: 'ProgressIndicator',
      id: 'components-progressindicator--default',
      globals: {
        theme: 'white',
      },
    });

    await expect(page.getByText('First step')).toBeVisible();

    page.getByText('First step').hover();

    await expect(page).toHaveNoACViolations('ProgressIndicator-onhover');
  });
  ```

Reply:
yeah I think you can do the same with the interactive story if necessary

Reply:
I added the new tests.

I'm checking if the prop `complete` (visited state) and `current` (active) are being added correctly, Because I don't see how to check using "ToHaveNoACViolations" for these two props. Only the hover made sense for me.

If you have an idea, let me know @francinelucca 😁